dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/Summary.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022-2023 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.task;
+
+import com.google.edwmigration.dumper.application.dumper.connector.ZonedInterval;
+import java.util.Optional;
+import java.util.function.BinaryOperator;
+
+public class Summary {
+
+  public static final BinaryOperator<Summary> COMBINER =
```

Comment:
Is this used only in tests?

Reply:
For now, yes. Overall `Summary` is just introduced in this PR without any final usage